prime-router/src/main/kotlin/ReportStreamFilter.kt

```diff
@@ -105,7 +106,6 @@ data class ReportStreamFilters(
         val defaultFiltersByTopic: Map<String, ReportStreamFilters> = mapOf(
             defaultCovid19Filters.topic to defaultCovid19Filters,
             defaultMonkeypoxFilters.topic to defaultMonkeypoxFilters,
-            defaultCsvFileTestFilters.topic to defaultCsvFileTestFilters,
```

Comment:
Not sure how this happened, or how it didn't break a test, but I'll add it back in and try to figure that out.